CMakeLists.txt

```diff
@@ -520,7 +520,7 @@ if(BUILD_TESTS)
     ADDITIONAL_ARGS
       --oe-binary
       ${OE_BINDIR}
-      --ledger-recovery-timeout
+      --ledger-read-timeout
```

Comment:
I am not sure this rename is good, but it's definitely incomplete (uses of the flag have not been updated), and it does not belong in this PR in any case.